.travis.yml

```diff
@@ -14,20 +14,29 @@ addons:
       - expect-dev # provides unbuffer utility
 
 stages:
+  - linting
   - test
 
 jobs:
   include:
+    - stage: linting
+      name: "pre-commit"
+      before_install:
+      install: pip install pre-commit
+      script: pre-commit run --all --show-diff-on-failure --verbose --color always
+      after_success:
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT="1"
+        - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="OCA/OCB"
+        - TESTS="1" ODOO_REPO="OCA/OCB"
+          EXCLUDE="fetchmail_incoming_log,fetchmail_notify_error_to_sender_test"
+
 env:
   global:
-    - VERSION="14.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
-    - MQT_DEP=PIP
+    - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
+    - COUNTRY="US"
```

Comment:
this is wrong

Reply:
@MiquelRForgeFlow Thanks for the review suggestions are implemented you can re-review it